pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py

```diff
@@ -20,9 +21,26 @@
 from packaging.version import Version as _Version
 from packaging.version import InvalidVersion
 from packaging.specifiers import SpecifierSet
+from typing import Optional, Any
 import collections
 import subprocess
 
+BLOCKLIST = {
```

Comment:
In the past I wanted to introduce a `passthru` attribute that indicated the updating strategy. That also gave the maintainers a way to say "Don't update my package". Not saying we should go that way, but it is an alternative.

Reply:
That sounds like a better approach, going to look into it.

Reply:
I decided for `passthru.denyBulkUpdate`. Unless anyone has a better name in mind.

Reply:
```diff
diff --git a/pkgs/development/python-modules/wandb/default.nix b/pkgs/development/python-modules/wandb/default.nix
index e1139ecf1b1..8764eca6ab9 100644
--- a/pkgs/development/python-modules/wandb/default.nix
+++ b/pkgs/development/python-modules/wandb/default.nix
@@ -164,6 +164,8 @@ buildPythonPackage rec {
     "wandb"
   ];
 
+  passthru.denyBulkUpdate = true;
+
   meta = with lib; {
     description = "A CLI and library for interacting with the Weights and Biases API";
     homepage = "https://github.com/wandb/wandb";
```

```
❯ maintainers/scripts/update-python-libraries --target minor --commit --use-pkgs-prefix pkgs/development/python-modules/wandb/default.nix
INFO:root:Updating packages...
WARNING:root:Path /home/hexa/git/nixos/python-updater/pkgs/development/python-modules/wandb/default.nix: Bulk update denied for wandb
INFO:root:Finished updating packages.
INFO:root:Using python3Packages. prefix for commits
INFO:root:Committing updates...
INFO:root:Finished committing updates
INFO:root:0 package(s) updated
```

Reply:
Do we want to limit it only to our bulk updates or also r-ryantm updates? Judging from the name it seems to be the former, and I think that's generally speaking also what is desired.

Reply:
I think allowing updates through r-ryantm is fine, since they go through a dedicated review. Not our business really.

Reply:
Not sure how to distinguish between the two though.

Reply:
If more than one package is affected, I now toggle a global knob. Not pretty, but it does the trick.